ios-base/DroidKaigi 2020/Announcements/Views/Cells/AnnouncementCell.swift

```diff
@@ -0,0 +1,41 @@
+import ios_combined
+import UIKit
+
+private extension Announcement.Type_ {
+    var iconImage: UIImage? {
+        switch self {
+        case .alert:
+            return #imageLiteral(resourceName: "warning_amber_24px")
+        case .feedback:
+            return #imageLiteral(resourceName: "assignment_24px")
+        case .notification:
+            return #imageLiteral(resourceName: "error_outline_24px")
+        default:
+            return nil
+        }
+    }
+}
+
+final class AnnouncementCell: UITableViewCell {
+    enum Constant {
+        static let dateFormatter: DateFormatter = {
+            let formatter = DateFormatter()
+            formatter.locale = Locale.current
+            formatter.calendar = Calendar.current
+            formatter.dateFormat = DateFormatter.dateFormat(fromTemplate: "dMMM HH:mm", options: 0, locale: Locale.current)
+            return formatter
+        }()
+    }
+
+    @IBOutlet weak var iconImageView: UIImageView!
+    @IBOutlet weak var publishedAtLabel: UILabel!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var contentLabel: UILabel!
+
+    func configura(_ announcement: Announcement) {
```

Comment:
typo. Please change function name to `configure` 🙏

Reply:
I fixed.

https://github.com/DroidKaigi/conference-app-2020/pull/628/commits/69dc4548170ba5a749e4e36e50a20a45156f6f5c